sklearn/cluster/tests/test_birch.py

```diff
@@ -16,12 +16,13 @@
 
 from sklearn.utils._testing import assert_almost_equal
 from sklearn.utils._testing import assert_array_equal
-from sklearn.utils._testing import assert_array_almost_equal
+from sklearn.utils._testing import assert_allclose
 
 
-def test_n_samples_leaves_roots():
+def test_n_samples_leaves_roots(global_dtype):
```

Comment:
This test does not really check numerical aspects of birch. I'm not sure it's relevant to parametrize it